tests/crm/test_crm.py

```diff
@@ -552,29 +552,49 @@ def test_crm_route(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_fro
 
 
 @pytest.mark.parametrize("ip_ver,nexthop", [("4", "2.2.2.2"), ("6", "2001::1")])
-def test_crm_nexthop(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, crm_interface, ip_ver, nexthop):
+def test_crm_nexthop(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, crm_interface, ip_ver, nexthop, ptfhost):
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
     asichost = duthost.asic_instance(enum_frontend_asic_index)
     RESTORE_CMDS["crm_threshold_name"] = "ipv{ip_ver}_nexthop".format(ip_ver=ip_ver)
-    nexthop_add_cmd = "{ip_cmd} neigh replace {nexthop} \
+    if duthost.facts["asic_type"] == "marvell":
+        vlan_mem_del_cmd = "config vlan member del 1000 Ethernet1"
+        vlan_mem_add_cmd =  "config vlan member add 1000 Ethernet1 -u"
+        if ip_ver == "4":
+           ptf_ip_add_cmd = "ifconfig eth1 {}/24 up".format(nexthop)
```

Comment:
How about use existing ptf method:
https://github.com/sonic-net/sonic-mgmt/blob/fec265e539769efa2eab2222cdf719f994116dee/tests/common/devices/ptf.py#L51
and 
https://github.com/sonic-net/sonic-mgmt/blob/fec265e539769efa2eab2222cdf719f994116dee/tests/common/devices/ptf.py#L90

Reply:
i dont find add_ip_to_dev method,
![image](https://user-images.githubusercontent.com/103626723/206656529-993326b0-2aeb-49d8-99ce-5b3d4a91dac7.png)

Reply:
Thanks, addressed review comments